Products/CMFPlomino/PlominoForm.py

```diff
@@ -1335,6 +1335,37 @@ def validation_errors(self, REQUEST):
                     errors=json.dumps({'success': True}))
 
 
+    security.declarePrivate('_get_js_hidden_subforms')
+    def _get_js_hidden_subforms(self, REQUEST, doc, validation_mode=False):
+        hidden_forms = []
+        hidewhens = json.loads(
```

Comment:
the method can be named `_get_js_hidden_fields`, I do not mind much, just fix this line https://github.com/instification/Plomino/blob/github-main/Products/CMFPlomino/PlominoForm.py#L1372 so we avoid useless serialize/deserialize sequences.